resources/assets/components/InboxItem/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { map } from 'lodash';
+
+class InboxItem extends React.Component {
+  displayImage(photo_url) {
+    if (photo_url == "default") {
```

Comment:
Where does `"default"` get set here? Is that an actual string in the database? If not, it could make sense to use [default props](https://facebook.github.io/react/docs/react-without-es6.html#declaring-prop-types-and-default-props) to automatically add a placeholder when the `file_url` is empty. 🐶